tests/unit/__init__.py

```diff
@@ -946,4 +962,7 @@ def setUp(self):
         self.content_start_confirm_address_title_en = 'Is this address correct?'
         self.content_start_confirm_address_title_cy = "Ydy'r cyfeiriad hwn yn gywir?"
 
+        self.content_request_address_not_listed_en = 'You need to call the Census customer contact centre'
```

Comment:
I suggest capitalising these 'Census Customer Contact Centre' to make it consistent with the text that appears beneath it.
```suggestion
        self.content_request_address_not_listed_en = 'You need to call the Census Customer Contact Centre'
```